src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Handlers/Fx2Nl/Fx2NlLanguageServerOperationHandler.cs

```diff
@@ -52,17 +52,17 @@ private Task PreHandleFx2NlAsync(LanguageServerOperationContext operationContext
             (scope) =>
             {
                 _preHandleCheckResult = scope?.Check(_fx2NlRequestParams.Expression) ?? throw new NullReferenceException("Check result was not found for Fx2NL operation");
-                _fx2NlParameters = GetFx2NlHints(scope, operationContext);
+                _fx2NlParameters = GetFx2NlParameters(scope, operationContext);
             }, cancellationToken);
         }
 
         /// <summary>
-        /// Gets the Fx2Nl hints.
+        /// Gets the Fx2Nl hints and range.

```

Comment:
range is part of hints/parameters. no need to mention it separately

Reply:
Ah but since we separating usage hints and range separately made sense to call this GetFx2NlParameters instead.